test/unittests/test_helpers.cpp

```diff
@@ -31,6 +31,13 @@ static constexpr size_t optionalDataSize =
 static_assert(optionalDataSize == sizeof(evmc_result_optional_storage), "");
 
 
+TEST(helpers, fnv1a)
```

Comment:
The commit log is wrong for this, it is about testing fnv1a.

Reply:
`git rebase -i` is planned.